rcongui/src/components/LogsView/logs.js

```diff
@@ -21,6 +22,49 @@ import ListItemText from "@material-ui/core/ListItemText";
 import FullscreenIcon from "@material-ui/icons/Fullscreen";
 import FullscreenExitIcon from "@material-ui/icons/FullscreenExit";
 
+const formatClass = (action, classes, highlightLogs) => {
+  // if the message is a chat message
+  if (highlightLogs) {
+    if (action.toLowerCase().includes("chat")) {
+      if (action.toLowerCase().includes("allies")) {
+        return classes.logsChatAllies;
+      }
+      if (action.toLowerCase().includes("axis")) {
+        return classes.logsChatAxis;
+      }
+    }
+    else if (action.toLowerCase().includes("admin")) {
+      return classes.logsAdmin;
+    }
+    else if (action.toLowerCase().includes("tk")) {
+      return classes.logsTK;
+    }
+    else if (action.toLowerCase().includes("match")) {
+      return classes.logsMatch;
+    }
+    else if (action.toLowerCase().includes("vote")) {
+      return classes.logsVote;
+    }
+    else switch (action.toLowerCase()) {
+      case "message":
+        return classes.logsMessage;
+      case "team kill":
+        return classes.logsTeamKill;
+      case "kill":
+        return classes.logsKill;
+      case "teamswitch":
+        return classes.logsTeamSwitch;
+      case "disconnected":
+        return classes.logsDisconnected;
+      case "connected":
+        return classes.logsConnected;
+      default:
```

Comment:
default case not really needed here. However, that would only be important, if we want to change the var name of the default color (and don't want to do it at two places).

Reply:
updated.